charmtools/build/tactics.py

```diff
@@ -170,18 +227,15 @@ class IgnoreTactic(Tactic):
     """
     @classmethod
     def trigger(cls, entity, target, layer, next_config):
-        """
-        Match if the given entity will be ignored by the next layer.
-        """
+        ""  # suppress inherited doc
```

Comment:
Hello future traveller.  If you're maintaining this code and stumble upon this PR on a quest to find out why the heck there's an empty string followed by an inline comment, you're just like me!

TIL that *no* docstring will inherit the docstring from base class methods. To prevent this, @johnsca has cleverly added empty docstrings where the inheritance is not wanted.